script/publish-docs

```diff
@@ -43,7 +37,9 @@ git fetch
 git checkout "$HEAD"
 cd ../..
 
-# Commit and publish
+# Configure Git and commit, and publish
+git config --global user.email "action@github.com"
+git config --global user.name "GitHub Action"
```

Comment:
Ah yeah, I ran into this before myself.

Reply:
Yeah, I forgot about it when we migrated to GA even though I had this same issue while I was working on #1100 and the script was originally running from GA. :rofl: